docs/README.rst

```diff
@@ -0,0 +1,223 @@
+What is Studio?
+===============
+
+|Hex.pm|
+
+Studio is a model management framework written in Python to help simplify and expedite your model building experience. It was developed to minimize the overhead involved with scheduling, running, monitoring and managing artifacts of your machine learning experiments. No one wants to spend their time configuring different machines, setting up dependencies, or playing archeology to track down previous model artifacts.
+
+Most of the features are compatible with any Python machine learning
+framework (`Keras <https://github.com/fchollet/keras>`__,
+`TensorFlow <https://github.com/tensorflow/tensorflow>`__,
+`scikit-learn <https://github.com/scikit-learn/scikit-learn>`__, etc) without invasion of your code;
+some extra features are available for Keras and TensorFlow.
+
+**Use Studio to:** 
+
+-  Capture experiment information- Python environment, files, dependencies and logs- without modifying the experiment code. 
+-  Monitor and organize experiments using a web dashboard that integrates with TensorBoard. 
+-  Run experiments locally, remotely, or in the cloud (Google Cloud or Amazon EC2) 
+-  Manage artifacts
+-  Perform hyperparameter search
+-  Create customizable Python environments for remote workers.
+
+Example usage
+-------------
+
+Start visualizer:
+
+::
+
+    studio ui
+
+Run your jobs:
+
+::
+
+    studio run myfile.py
+
+You can see results of your job at http://127.0.0.1:5000. Run
+``studio {ui|run} --help`` for a full list of ui / runner options
+
+Installation
+------------
+
+Installation Packaging
+~~~~~~~~~~~~~~~~~~~~~~
+
+pip install studioml from the master pypi repositry:
+
+::
+
+    pip install studioml
+
+or, install the source and development environment for StudioML from the git project directory:
+
+::
+
+    git clone https://github.com/studioml/studio && cd studio && pip install -e .
+
+A setup.py is included in the top level of the git repository that
```

Comment:
in the future, I think we should put the rest of this section into a separate doc file (it feels a bit too technical for the README)

Reply:
Agreed. Coming up later today: stripped down basic README.